ui.frontend/src/main/webpack/site/js/components/buildFacet.ts

```diff
@@ -117,13 +118,17 @@ BuildFacetOption): HTMLDivElement => {
           tabUrl,
           searchValue,
           searchResultsElement: currentTab,
-          onLoadMoreButtonClick: undefined,
+          onLoadMoreClick: undefined,
           queryParameterName,
         })
         currentTab.appendChild(loadMoreButton)
       }
     }
   })
+  // eslint-disable-next-line @typescript-eslint/no-misused-promises
+  facetInput.addEventListener('change', () => {
+    facetInput.dispatchEvent(onFacetsSelect)
+  })
```

Comment:
As discussed, no need to have another addEventListener. Just dispatch the event directly wherever addEventListener was created.